wordle-web/src/components/AppBar.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <v-app-bar id="toolbar">
+    <v-toolbar-title>
+      <div @click="router.push('/')" id="toolbar-title">
+        <v-icon icon="mdi-book-open-page-variant" />
+        The Good Word
+      </div>
+    </v-toolbar-title>
+
+    <v-btn @click="toggleMusic">
+      <v-icon size="large">{{ isPaused ? 'mdi-volume-high' : 'mdi-volume-mute' }}</v-icon>
```

Comment:
I think I would switch these icons. Consider how the windows icon is used... it shows as muted when the sound is muted and as on when the sound is on
![image](https://user-images.githubusercontent.com/43832670/236657297-330dcb48-83cf-470d-9275-6e1de8109b76.png)